stable/appmesh-controller/README.md

```diff
@@ -48,9 +50,87 @@ kubectl apply -k github.com/aws/eks-charts/stable/appmesh-controller//crds?ref=m
 
 Install the App Mesh CRD controller:
 
+### EKS on Fargate
```

Comment:
Nit: I think the `Regular Kubernetes distribution` section should come first, since its the simplest and probably will be the most common use case.

Reply:
done, please take a look. thanks